### ISSUE TITLE
"Copy", "Paste" commands not functional on text from translated webpage

### ISSUE THREAD

**Issue by alexandra-martin** [mozilla/firefox-voice#1117]
### Prerequisites:
Mic permission are enabled.
"Allow" or "Don't Allow" button from “Allow Firefox Voice to Collect Voice Transcriptions” pop-up needs to be clicked.
A webpage is translated to any language and made the active tab.

### STR:
1. Select text from the webpage. 
2. Use the shortcut or click on the mic icon from the browser toolbar.
3. Say or write the "Copy" command.
4. Open a new tab.
5. Use the shortcut or click on the mic icon from the browser toolbar.
6. Say or write the "Paste" command.

### Expected result:
The relevant copied text is pasted in the URL bar and a Google search is performed.

### Actual result:
Text that was copied before performing the "Copy" command is pasted in the URL bar and a Google search is performed.

### Notes:
Reproduced on Mac 10.14.6 and Win10x64 with Firefox Release 73.0.1 and Nightly 75.0a1 (2020-02-27).
If no text was copied before performing the "Copy" command, the "Paste" command will perform an empty Google search. (2nd gif)

![2](https://user-images.githubusercontent.com/54401091/75537317-787a7a00-5a1f-11ea-8c17-44d31ff4754c.gif)

![3](https://user-images.githubusercontent.com/54401091/75537820-559c9580-5a20-11ea-9def-904e957b093d.gif)

**Comment by shjang7** [comment]
Hello, I am an applicant for the Outreachy internship, can I take this issue for my first contribution?

**Comment by AemieJ** [comment]
Hello, I am an applicant for the Outreachy internship, and I would love to take this issue as my first contribution. Is it possible that I could use this issue as my first contribution?

**Comment by naima-shk** [comment]
Hi . I m an Outreachy applicant  , can i take this one please ?

**Comment by Sanchibisht29** [comment]
hi am an Outreachy applicant, can i pls take this issue?

**Comment by saucekode** [comment]
Hello, an outreachy applicant here. Please, I would like to take this issue. Thank you.

**Comment by saucekode** [comment]
@alexandra-martin please could you rephrase the actual result. Can't seem to understand the statement. Thank you so much.

**Comment by saucekode** [comment]
@alexandra-martin this actually works but it works later but not immediately like an english text

**Comment by ianb** [comment]
OK, I think I'm hearing this isn't a bug, so I'll close it and @alexandra-martin can reopen if that seems incorrect.

**Comment by alexandra-martin** [comment]
Retried on Mac 10.14 and Win10x64 with Firefox Nightly 76.0a1 (2020-03-09) with the steps provided and I can still reproduce the issue. 
In my case, if a page is translated from the beginning or is already in any language, (without using "Translate" commands) then everything is fine.
![6](https://user-images.githubusercontent.com/54401091/76311253-2ff17500-62d9-11ea-8d69-a5441914ee70.gif)

But if a "Translate" command is made on a page, then a paragraph is selected and copied with the "Copy" command, the pasted text using the "Paste" command is not the text that was intended to be pasted.
![7](https://user-images.githubusercontent.com/54401091/76311499-c6259b00-62d9-11ea-9eeb-4e93ff85f299.gif)

@saucekode can you give me your example, or your process, so I can try as well?

**Comment by saucekode** [comment]
Yes I do understand what you say. But the issue is that, it works but it's
slow. That means it works later. Perhaps you could fix that. Thanks.

**Comment by alexandra-martin** [comment]
@saucekode thanks for the quick response. But when you say that it works later, do you refer that the relevant text is pasted, if a second "Paste" command is made? Cause even after multiple "Paste" commands, the same text is displaying for me.

**Comment by shjang7** [comment]
@alexandra-martin As I understand this issue, 
- It works well when simply copy and paste.
- But, it doesn't work when 1. translate 2. copy 3. paste.
Am I correct understanding this issue?

By the way, do you have any online demo link about this program?
I want to test it on my local machine and where I could find the instruction to install this program?
Is it working in the master branch? and $ npm install doesn't work, how do I reach to you about the question about installing?

**Comment by ianb** [comment]
I'm wondering if this is actually the same issue as #1205 – something on these sites is keeping the copy functionality from working in general.
